packages/frontend/src/services/NonFungibleTokens.js

```diff
@@ -45,30 +55,54 @@ export default class NonFungibleTokens {
             });
         }
         // TODO: Separate Redux action for loading image
-        tokens = await Promise.all(tokens.filter(({ metadata }) => !!metadata).map(async ({ metadata, ...token }) => {
-            const { media } = metadata;
-            let mediaUrl;
-            if (!media.includes('://')) {
-                if (base_uri) {
-                    mediaUrl = `${base_uri}/${media}`;
-                } else {
-                    mediaUrl = media.startsWith('data:image') ? media : `https://cloudflare-ipfs.com/ipfs/${media}`;
-                }
-            } else {
-                mediaUrl = media;
-            }
-
-            return {
-                ...token,
-                metadata: {
-                    ...metadata,
-                    mediaUrl
-                }
-            };
-        }));
+        tokens = tokens
+          .filter(({ metadata }) => !!metadata)
+          .map(token => mapTokenMediaUrl(token, base_uri));
 
         return tokens;
     }
+
+    static transfer = async ({ accountId, contractId, tokenId, receiverId }) => {
+        const account = await wallet.getAccount(accountId);
+        const nftTransferDeposit = 1;
```

Comment:
nit: can we please add this as `NFT_TRANSFER_DEPOSIT` at the top of the file below `NFT_TRANSFER_GAS` :)

Reply:
done